### DISCUSSION TITLE
How to delete a foreign key column?

### DISCUSSION THREAD

**Discussion by jhh78** [pocketbase/pocketbase#4358] [Q&A] [answered]
greetings.
Assume you have the following table structure:

![image](https://github.com/pocketbase/pocketbase/assets/51219166/af3e9f32-bfb0-47ac-bad4-7136b8f1e8de)

When the data in the member table is deleted, I also want to delete the data in the message table connected to it.

**Comment by pnmcosta** [comment, marked as answer]
This is built in, on your message relation field, make sure "Cascade delete" is set to true:
![image](https://github.com/pocketbase/pocketbase/assets/550684/1e0b4cc5-a2e8-4ce2-8790-3f4d71e0bbff)
